packages/karma/karma.spec

```diff
@@ -1,10 +1,10 @@
 Name:     karma
 Summary:  karma
 Release:  1%{?dist}
-Version:  0.68
+Version:  %{_software_version}
 License:  apache-2.0
 Group:    System Environment/Base
-Source:   https://github.com/prymitive/karma/releases/download/v0.68/karma-linux-amd64.tar.gz
+Source:   https://github.com/prymitive/karma/releases/download/v%{_software_version}/karma.tar.gz
```

Comment:
The name of the source file mismatch the name or the wget target below.

Reply:
Solved